libraries/Microsoft.BotKit.Adapters.Slack/AdapterError.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright(c) Microsoft Corporation.All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.BotKit.Adapters.Slack
+{
+    public class AdapterError
+    {
+        public AdapterError()
```

Comment:
Is an empty ctor needed? I think this could be deleted.

Reply:
Feedback applied. Thanks!